.travis.yml

```diff
@@ -69,7 +72,7 @@ before_script:
 
 script:
   - sh -c "if [ '$DEFAULT' = '1' ]; then phpunit --stderr; fi"
-  - sh -c "if [ '$PHPCS' = '1' ]; then vendor/bin/phpcs -p --extensions=php --standard=psr2 --ignore=tests/bootstrap.php,tests/comparisons/*,tests/test_app/* ./src ./tests ; fi"
+  - sh -c "if [ '$PHPSTAN' = '1' ]; then composer require --dev phpstan/phpstan:^0.6 && vendor/bin/phpstan analyse -l 0 src; fi"
```

Comment:
Try higher rule levels just to see what kinds of issues PHPStan comes up with :) But to start with level 0 is fine, even recommended.